plone/autoform/tests/test_base.py

```diff
@@ -119,3 +119,10 @@ def test_calc_field_move_override(self):
         self.assertIn('z', rules)
         self.assertNotIn('a', rules)
         self.assertNotIn('c', rules)
+
+    def test_updateFieldsFromSchemata(self):
+        from plone.autoform.base import AutoFields
+        autofields = AutoFields()
+        autofields.request = {}
+        autofields.updateFieldsFromSchemata()
+        # (the assertion is that is doesn't break)
```

Comment:
then you could do a ``self.assertTrue(autofields.updateFieldsFromSchemata())`` ?